nx-X11/lib/X11/Imakefile

```diff
@@ -891,13 +893,7 @@ OBJS3 = \
 	xim_trans.o \
 	$(XKBOBJS) $(LCOBJS)
 
-SHAREDLCDEPSYMSRCS = XlcDL.c XlcSL.c
-
-SHAREDI18NSTATICSRCS = \
-        XDefaultIMIF.c XDefaultOMIF.c
-
 STATICLCDEPSYMSRCS =
-
```

Comment:
dropping ``STATICLCDEPSYMSRCS =`` completely soon, so also dropping that empty line